server/schedulers/hot_region.go

```diff
@@ -1099,33 +1104,44 @@ func (h *hotScheduler) GetPendingInfluence() map[uint64]*Influence {
 	return ret
 }
 
-// calcPendingWeight return the calculate weight of one Operator, the value will between [0,1]
-func (h *hotScheduler) calcPendingWeight(op *operator.Operator) float64 {
-	if op.CheckExpired() || op.CheckTimeout() {
-		return 0
-	}
+// calcPendingInfluence return the calculate weight of one Operator, the value will between [0,1]
+func (h *hotScheduler) calcPendingInfluence(op *operator.Operator, maxZombieDur time.Duration) (weight float64, needGC bool) {
```

Comment:
Why do we need `needGC` flag?

Reply:
This is ported from the previous GC strategy

Reply:
I think maybe we can tell it apart through whether it is 0 or 1?

Reply:
It is now used in statistics and judging duplicates. The `weight` is used for statistics, and the `GC` is used for judging duplicates. This PR does not want to adjust the strategy.

Reply:
For the operator which reaches the end status, perhaps we should GC it to avoid unnecessary influence.

Reply:
I am not sure if it is the right way. Once the operator is canceled, it needs to wait for 30s to recreate the operator if it's still hot.

Reply:
no need to use float